DataDog/datadog-agent
# Celene/static tags collector

### What does this PR do?
Adds a simple FetchOnly static tags collector that can be used for tags that do not change after start, e.g. those from environment variables.

Adds `DD_TAGS` env var extract to static collector and removes this functionality from the ECS fargate collector.

### Motivation

This allows `DD_TAGS` to be applied in environments where specialized collectors were not present such as in EKS Fargate.

### Describe your test plan

- Ensure `DD_TAGS` are applied when monitored pods and containers are started or cycled
- Ensure `DD_TAGS` are applied in ECS and EKS Fargate
- Ensure `DD_TAGS` are part of host tags and not output of `agent tagger-list` for non-EKS clusters
